group-coordinator/src/main/java/org/apache/kafka/coordinator/group/Group.java

```diff
@@ -166,4 +166,20 @@ void validateOffsetFetch(
      * @return true if the state includes, false otherwise.
```

Comment:
I would move this one to common too.

Reply:
Didn't move it as the interface is being used in ClassicGroup as well, hence just moved the one's which are common for Share and Comsumer group to `modern`.